commander/src/bootstrapping/templates/lisk-template-ts/templates/app/test/unit/modules/module.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+// import * as modules from '../../../src/app/modules'
+
+describe('<%= moduleClass %>Module', () => {
+	beforeEach(() => {});
+
+	describe('constructor', () => {
+		it.todo('should have valid id');
+		it.todo('should have valid name');
+	});
+
+	describe('beforeBlockApply', () => {});
+	describe('afterBlockApply', () => {});
+	describe('beforeTransactionApply', () => {});
+	describe('afterTransactionApply', () => {});
+	describe('afterGenesisBlockApply', () => {});
```

Comment:
These empty blocks is causing linting error in the generated app. Either we add some implementation or remove empty functions in definitions.

Reply:
ts-morph lib for import statement uses double quotes, `import { Nft } from "./modules/nft";` this is the only lint error which is not fixed